contrib/!irc/erc/README.md

```diff
@@ -26,6 +26,8 @@ Layer for [ERC IRC chat][].
   (using [erc-view-log][])
 - YouTube videos Thumbnails inline (using [erc-yt][])
 - Social Graph for ERC messages (using [erc-social-graph][])
+- ERC yank, if pasting more than 5 lines, create a gist for it. (needs to have
+  `git-enable-github-support` set to `t` in the git layer)
```

Comment:
This variable does not exist anymore, all the github related packages are now in the `github` layer.